src/Merchants/Migration/Migration1585212510DoubleOptIn.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Production\Merchants\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+class Migration1585212510DoubleOptIn extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1585212510;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->insert('system_config', [
```

Comment:
I guess we cannot do this anymore. Cause we will have also customers now. So we have to enable that only fly, when a merchant will be registered